README.md

```diff
@@ -1,3 +1,74 @@
-# CFEngine Bot, Tom
+# Northern.tech Bot, Tom
 
 I'm your friendly neighbourhood bot, Tom.
+I trigger pull requests when someone mentions me in a GitHub Pull Request.
+
+## Usage
+
+## Command line
```

Comment:
Should Usage and Command line be different levels of hash-tags?

Reply:
Yup, don't know why that happened.